ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -51,7 +51,8 @@ object KeyValueConsumption {
               Some(entry.getPackageUploadEntry.getSourceDescription)
             else None,
             parseLedgerString("ParticipantId")(entry.getPackageUploadEntry.getParticipantId),
-            recordTime
+            recordTime,
+            TracingInfo("")
```

Comment:
That one is on the way from the ledger to the ledger-aapi-server. They should be filled with info relayed through the system from the original call. Because, at the moment I have only introduced an interface, plumbing isn't there yet to pass that between all the components

Reply:
Perhaps something that makes this obvious, like `TracingInfo.NotPlumbedYet`?

It can be defined as `TracingInfo("")` under the hood.